src/frontend/package.json

```diff
@@ -16,10 +16,11 @@
     "type-check": "tsc --noEmit",
     "lint": "concurrently \"node_modules/.bin/eslint **/*.{ts,tsx} --quiet --fix\" \"node_modules/.bin/stylelint --fix '**/*.{js,ts,tsx,css}'\"  \"npm run type-check\"",
     "lint-ci": "concurrently \"node_modules/.bin/eslint **/*.{ts,tsx}\" \"node_modules/.bin/stylelint '**/*.{js,ts,tsx,css}'\"  \"npm run type-check\"",
+    "lint-e2e": "concurrently \"node_modules/.bin/eslint --fix e2e/**/*.ts\" \"node_modules/.bin/stylelint --fix 'e2e/**/*.ts'\"  \"npm run type-check\"",
     "pl:staging": "NODE_ENV=staging npx playwright test --headed -c ./e2e/setup/staging.config.ts",
-    "pl:local": "NODE_ENV=local npx playwright test --headed -c ./e2e/setup/staging.config.ts",
-    "pl:staging:headless": "npx playwright test --headless -c ./e2e/setup/staging.config.ts",
-    "pl:staging:debug": "npx playwright test --debug -c ./e2e/setup/staging.config.ts",
+    "pl:local": "NODE_ENV=local npx playwright test --headed -c ./e2e/setup/local.config.ts",
+    "pl:staging:headless": "NODE_ENV=local NODE_ENV=staging npx playwright test --headless -c ./e2e/setup/staging.config.ts",
```

Comment:
you have 2 `NODE_ENV` s here

Reply:
these were actually charles modifications but i have removed them now